cns/restserver/homeazmonitor.go

```diff
@@ -126,6 +126,12 @@ func (h *HomeAzMonitor) Populate(ctx context.Context) {
 				h.update(returnCode, returnMessage, cns.HomeAzResponse{IsSupported: true})
 				return
 
+			case http.StatusNotFound:
+				returnMessage := fmt.Sprintf("[HomeAzMonitor] region does not support AZs probably, failed with StatusCode: %d, error: %v", apiError.StatusCode(), err)
```

Comment:
nit: remove 'probably'
instead of saying this 'failed' ( Which is not the case ) IMO we should simply indicate that NMA returned apiError.StatusCode()

Reply:
Addressed